upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -365,11 +365,8 @@ func (c *ApplyClusterCmd) Run() error {
 	case fi.CloudProviderVSphere:
 		{
 			vsphereCloud := cloud.(*vsphere.VSphereCloud)
-			region = vsphereCloud.Region
-
-			//if !AlphaAllowGCE.Enabled() {
-			//	return fmt.Errorf("GCE support is currently alpha, and is feature-gated.  export KOPS_FEATURE_FLAGS=AlphaAllowGCE")
-			//}
+			// TODO: map region with vCenter cluster, or datacenter, or datastore?
+			region = vsphereCloud.Cluster
```

Comment:
GCE also has a feature flag https://github.com/kubernetes/kops/blob/f53070455051dca489632b9c47bc4ac6333e5c2f/upup/pkg/fi/cloudup/apply_cluster.go#L57. I think it makes sense to prevent use of vSphere as cloud provider till we are ready for people to try it out. Let's discuss this separately though.